### PR TITLE
update

### PR DESCRIPTION
**Pull request by ilovemane**

### CODE REVIEW

File: R/classSummarisedResult.R [hunk 1]
```diff
@@ -161,7 +163,19 @@ constructSummarisedResult <- function(x, set, call = parent.frame()) {
   }
 
   if (is.null(set)) {
-    set <- x |> dplyr::select("result_id") |> dplyr::distinct()
+    set <- x |>
+      dplyr::select("result_id") |> dplyr::distinct()
```
Comment (catalamarti) [inline R/classSummarisedResult.R:167]: one line or three lines, my TOC doesn't allow 2

File: R/classSummarisedResult.R [hunk 2]
```diff
@@ -161,7 +163,19 @@ constructSummarisedResult <- function(x, set, call = parent.frame()) {
...
+  }
+
+  requiredSettingsColumns <-
+    c("result_type" , "package_name", "package_version")
+  for (name in requiredSettingsColumns) {
+    if (!(name %in% xCol) & !(name %in% names(set))) {
```
Comment (catalamarti) [inline R/classSummarisedResult.R:173]: why is xCol needed?

File: R/classSummarisedResult.R [hunk 3]
```diff
@@ -544,5 +558,6 @@ emptySummarisedResult <- function(settings = NULL) {
     rlang::rep_named(list(character())) |>
     dplyr::as_tibble() |>
     dplyr::mutate("result_id" = as.integer()) |>
-    newSummarisedResult(settings = settings)
+    newSummarisedResult(settings = settings) |>
+    suppressWarnings()
```
Comment (catalamarti) [inline R/classSummarisedResult.R:562]: should we instead of suppressWarnings add them as a tibble?

File: tests/testthat/test-emptyObjects.R [hunk 4]
```diff
@@ -49,15 +49,18 @@ test_that("you can create empty objects",{
     cohortColumns("cohort_attrition") %in% (cdm$my_cohort |> attrition() |> colnames())
   ))
   expect_error(cdm <- emptyCohortTable(cdm = cdm, name = "person"))
-
+  settings(emptySummarisedResult())
```
Comment (catalamarti) [inline tests/testthat/test-emptyObjects.R:52]: ?

**Review by catalamarti — APPROVED** [review]
thanks @ilovemane very minor things

File: R/classSummarisedResult.R (hunk 2 above)
Comment (ilovemane) [reply R/classSummarisedResult.R:173]: some columns will get moved to settings from the initial input x.

**Pull request merged**